packages/paste-core/components/popover/src/Popover.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, BoxProps, safelySpreadBoxProps} from '@twilio-paste/box';
+import {NonModalDialogPrimitive} from '@twilio-paste/non-modal-dialog-primitive';
+import {PopoverArrow} from './PopoverArrow';
+import {PopoverContext} from './PopoverContext';
+
+export interface PopoverProps {
+  'aria-label': string;
+  children: React.ReactNode;
+  tabIndex?: number;
+}
+
+const StyledPopover = React.forwardRef<HTMLDivElement, BoxProps>(({style, ...props}, ref) => {
+  return (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      backgroundColor="colorBackgroundBody"
+      borderStyle="solid"
+      borderWidth="borderWidth10"
+      borderColor="colorBorderLight"
+      borderRadius="borderRadius20"
+      boxShadow="shadowCard"
+      maxWidth="size30"
+      minWidth="size30"
+      zIndex="zIndex10"
+      _focus={{outline: 'none'}}
+      style={style}
+      ref={ref}
+    />
+  );
+});
+
+const Popover = React.forwardRef<HTMLDivElement, PopoverProps>(({children, ...props}, ref) => {
+  const popover = React.useContext(PopoverContext);
+  return (
+    <NonModalDialogPrimitive {...(popover as any)} {...props} as={StyledPopover} ref={ref}>
```

Comment:
Is there any way you can think of, of positioning a close x button in the top right and making that always render?

I’d like to make it a rule that all non-modal dialogs should have a visual dismiss action for dictation users, and if you can come up with a way to make it look nice, not get in the way of other people’s content, that would be great

Reply:
I was able to work this out: 
![image](https://user-images.githubusercontent.com/1350081/84530384-4d4f2a00-aca0-11ea-93fd-b961f579ad49.png)